environments/standalone/inventory/group_vars/all/variables.yml

```
@@ -0,0 +1,38 @@
+---
+kubeconfig_path: "~/Documents/StackHPC/azimuth/leafcloud_kubeconfig"
+
+configure_system_resources: false # setup for a custom CA and setting different defaults for k3s etc
+install_k3s: false
+install_cli_tools: false # useful on a VM but could overwrite existing tools on localhost
+slurp_k3s_kubeconfig: false # will overwrite existing kubeconfig
+ingress_controller_enabled: false
+
+# assumes the deploy host is also hosting the cluster, this may need to be overridden
+# infra_external_ip: 45.135.59.103
+# infra_external_ip: "{{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }}" # replace with cluster IP
+# ingress_base_domain: "apps.{{ infra_external_ip | replace('.', '-') }}.sslip.io"
+
+# Disable HTTPS and certificate verification everywhere to avoid certificate issues (false by default)
+ingress_tls_enabled: false
+
+azimuth_current_cloud_name: k3s-azimuth
```

Nit: We probably shouldn't use k3s in the name here
```suggestion
azimuth_current_cloud_name: standalone
```